central/pruning/pruning_test.go

```diff
@@ -1133,7 +1133,7 @@ func (s *PruningTestSuite) TestAlertPruning() {
 			for _, deployment := range c.deployments {
 				require.NoError(t, deployments.UpsertDeployment(ctx, deployment))
 			}
-			all, err := alerts.Search(ctx, getAllAlerts())
+			all, err := alerts.Search(ctx, getAllAlerts(), true)
```

Comment:
The pruner should get all alerts.  So recommend `false`.  This is one of the use cases we hit that spawned this.  I eventually worked around it by using a different method.